java/src/jmri/jmrit/display/MultiSensorIconAdder.java

```diff
@@ -105,10 +105,6 @@ protected void makeIcons(CatalogTreeNode n) {
         }
     }
 
-    /**
-     * Only called from MultiSensorIcon popup
-     *
-     */
```

Comment:
Instead of removing block JavaDoc comments like this (ones on protected/private/package members that aren't really interface definitions), would it be OK to turn them into regular block comments that start with `/*`:

```
/*
 * Only called from MultiSensorIcon popup
 *
 */
```